src/TimingSegments.h

```diff
@@ -82,7 +82,7 @@ struct TimingSegment
 	float GetBeat() const { return NoteRowToBeat(m_iStartRow); }
 	void SetBeat( float fBeat ) { SetRow( BeatToNoteRow(fBeat) ); }
 
-	virtual RString ToString(int /* dec */) const
+	virtual RString ToString(int dec = 0) const
```

Comment:
can we just delete this argument if its unused? Or do subclasses make use of it?